### DISCUSSION TITLE
Access an OPC UA Server that is running in a mobile network

### DISCUSSION THREAD

**Discussion by deretive** [FreeOpcUa/python-opcua#1374] [Q&A] [answered]
Is there any possibility to access an OPC UA Server that is connected to the internet through GSM?

Thanks in advance

**Comment by AndreasHeine** [comment, marked as answer]
with increased timeouts propably! but opc ua is a heavy protocol and not particularly developed for those usecases.

**Reply by deretive** [reply]
Is there anything special in a code to realize this?
Once I was able to do so, but were not able to reproduce the connection.
Could the NAT of the GSM provider block the connection? As Endpoint I use my IP4 address.

**Reply by AndreasHeine** [reply]
not sure how your provider handles that, typically a vpn tunnle is used to access remote devices in the field.

**Reply by deretive** [reply]
Thanks for your answer!
